docs/syntax.md

```diff
@@ -0,0 +1,363 @@
+  ![##ff66cc](https://via.placeholder.com/15/ff66cc/000000?text=+)
+  ![##0066ff](https://via.placeholder.com/15/0066ff/000000?text=+)
+  ![##ff0000](https://via.placeholder.com/15/ff0000/000000?text=+)
+  ![##000000](https://via.placeholder.com/15/000000/000000?text=+)
+  ![##8000ff](https://via.placeholder.com/15/8000ff/000000?text=+)
+  ...
+
+- `IEC` for [IEC 62](https://en.wikipedia.org/wiki/Electronic_color_code#Color_band_system) ("ROY G BIV")
+
+  ![##895956](https://via.placeholder.com/15/895956/000000?text=+)
+  ![##ff0000](https://via.placeholder.com/15/ff0000/000000?text=+)
+  ![##ff8000](https://via.placeholder.com/15/ff8000/000000?text=+)
+  ![##ffff00](https://via.placeholder.com/15/ffff00/000000?text=+)
+  ![##00ff00](https://via.placeholder.com/15/00ff00/000000?text=+)
+  ![##0066ff](https://via.placeholder.com/15/0066ff/000000?text=+)
+  ![##8000ff](https://via.placeholder.com/15/8000ff/000000?text=+)
+  ![##999999](https://via.placeholder.com/15/999999/000000?text=+)
+  ![##ffffff](https://via.placeholder.com/15/ffffff/000000?text=+)
+  ![##000000](https://via.placeholder.com/15/000000/000000?text=+)
+  ...
+
+- `TEL` and `TELALT`  for [25-pair color code](https://en.wikipedia.org/wiki/25-pair_color_code)
+- `T568A` and `T568B` for [TIA/EIA-568](https://en.wikipedia.org/wiki/TIA/EIA-568#Wiring) (e.g. Ethernet)
+- `BW` for alternating black and white
+
+
+## Images
+
+Both connectors and cables accept including an image with a caption within their respective nodes.
+
+```yaml
+image:
+  src: <path>        # path to the image file
+  # optional parameters:
+  caption: <str>     # text to display below the image
+  width: <int>       # range: 1~65535; unit: ???
+  height: <int>      # range: 1~65535; unit: ???
+  fixedsize: <bool>  # ???
+  scale: <str>       # scaling behavior; possible values:
+                     # false   does not scale the image
+                     # true    scale image proportionally
+                     #         to fit within given max. width and height
+                     # width   scale image proportionally to given width
+                     # height  scale image proportionally to given height
+                     # both    resize to given width and height
+```
+
+
+## Multiline strings
+
+Connectors accept multiline strings in the `type`, `subtype` and `notes` attributes.
+Cables accept multiline strings in the `type` and `notes` attributes.
+
+### Method 1
+
+By using `|`, every following indented line is treated as a new line.
+
+```yaml
+attribute: |
+  This is line 1.
+  This is line 2.
+```
+
+### Method 2
+
+By using double quoted strings, `\n` within the string is converted to a new line.
+
+```yaml
+attribute: "This is line 1.\nThis is line 2."
+```
+
+Plain (no quotes) or single quoted strings do not convert `\n`.
+
+See [yaml-multiline.info](https://yaml-multiline.info/) for more information.
+
+## Inheritance
+
+Add link to YAML spec.
```

Comment:
Remember adding a link here.

Reply:
I'll add a link to [this page](https://blog.daemonl.com/2016/02/yaml.html) which explains YAML anchors very concisely.
Since it's supposed to be a WireViz syntax description, and not a YAML tutorial, I'll keep it short and sweet for now, with a reference to demo02 for an example. The `--prepend-file` function is not really part of the syntax, so I won't include it here.

My hope is that in the future, someone with more writing skills than me can contribute a full, from-start-to-finish WireViz tutorial for first-time users with no previous experience with YAML, explaining all these features in a friendly and understandable way. But that is an issue for another day ;)